### PR TITLE
add concat and batchnorm

### PR DESCRIPTION
**Pull request by SujieZhu**
## What changes were proposed in this pull request?
I add the Concat and BatchNorm Layer to the TFToBigDL.scala

And add dataFormat in the Object  TFToBigDL
(Please fill in changes proposed in this patch)

## How was this patch tested?

(Please explain how this patch was tested. E.g. unit tests, integration tests, manual tests)
(If it is possible, please attach a screenshot; otherwise, remove this)

## Related links or issues (optional)
fixed https://github.com/intel-analytics/BigDL/issues/XXX

### CODE REVIEW

File: spark/dl/src/main/scala/com/intel/analytics/bigdl/utils/TFToBigDL.scala [hunk 1]
```diff
@@ -297,6 +297,91 @@ object IdentityTF extends TFToBigDL {
   }
 }
 
+object BatchNormTF extends TFToBigDL{
+  private val graph = {
+    val nodeInput = new Node("*")
```
Comment (yiheng) [inline spark/dl/src/main/scala/com/intel/analytics/bigdl/utils/TFToBigDL.scala]: remove the new

File: spark/dl/src/main/scala/com/intel/analytics/bigdl/utils/TFToBigDL.scala [hunk 2]
```diff
@@ -309,6 +394,10 @@ object TFToBigDL {
 
   private var endian = ByteOrder.LITTLE_ENDIAN
 
+  var dataFormat : String = "NHWC"
```
Comment (yiheng) [inline spark/dl/src/main/scala/com/intel/analytics/bigdl/utils/TFToBigDL.scala]: private?

File: spark/dl/src/main/scala/com/intel/analytics/bigdl/utils/TFToBigDL.scala [hunk 3]
```diff
@@ -309,6 +394,10 @@ object TFToBigDL {
...
+
+  def dataNCHW : Unit = dataFormat = "NCHW"
```
Comment (yiheng) [inline spark/dl/src/main/scala/com/intel/analytics/bigdl/utils/TFToBigDL.scala]: java doc

Comment (yiheng) [inline spark/dl/src/main/scala/com/intel/analytics/bigdl/utils/TFToBigDL.scala]: change string to case class

**Comment by SujieZhu** [comment]
I have modified the matchGraph and add inception_v3 test

**Pull request merged**